addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11430,7 +11430,7 @@ msgid "No video files found in this path!"
 msgstr ""
 
 msgctxt "#20350"
-msgid "votes"
+msgid "%.1f (%i votes)"
```

Comment:
This might be confusing for translators I think?
I think there is some kind of comment you can add here, that will show up when doing translations